### PR TITLE
Move Cassandra raw frame methods to a module

### PR DESCRIPTION
**Pull request by conorbros**

### CODE REVIEW

File: shotover-proxy/tests/helpers/mod.rs [hunk 1]
```diff
@@ -28,7 +28,7 @@ impl ShotoverManager {
     pub fn from_topology_file(topology_path: &str) -> ShotoverManager {
         let opts = ConfigOpts {
             topology_file: topology_path.into(),
-            config_file: "tests/helpers/config.yaml".into(),
+            config_file: "config/config.yaml".into(),
```
Comment (rukai) [inline shotover-proxy/tests/helpers/mod.rs:31]: is this a mistake?

Comment (conorbros) [reply shotover-proxy/tests/helpers/mod.rs:31]: Yes, I'll fix

**Review by rukai — APPROVED** [review]
this separation makes sense to me

**Review by Claude-at-Instaclustr — APPROVED** [review]
LGTM

**Pull request merged**